Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -179,22 +186,32 @@ void MemoryWidget::CreateWidgets()
   address_space_layout->setSpacing(1);
 
   // Data Type
-  auto* datatype_group = new QGroupBox(tr("Data Type"));
-  auto* datatype_layout = new QVBoxLayout;
-  datatype_group->setLayout(datatype_layout);
-
-  m_type_u8 = new QRadioButton(tr("U&8"));
-  m_type_u16 = new QRadioButton(tr("U&16"));
-  m_type_u32 = new QRadioButton(tr("U&32"));
-  m_type_ascii = new QRadioButton(tr("ASCII"));
-  m_type_float = new QRadioButton(tr("Float"));
-
-  datatype_layout->addWidget(m_type_u8);
-  datatype_layout->addWidget(m_type_u16);
-  datatype_layout->addWidget(m_type_u32);
-  datatype_layout->addWidget(m_type_ascii);
-  datatype_layout->addWidget(m_type_float);
-  datatype_layout->setSpacing(1);
+  auto* displaytype_group = new QGroupBox(tr("Display Type"));
+  auto* displaytype_layout = new QVBoxLayout;
+  displaytype_group->setLayout(displaytype_layout);
+
+  m_display_combo = new QComboBox;
+  m_display_combo->setMaxVisibleItems(20);
```

Comment:
Is this necessary?

Reply:
The combobox will have a scroll bar if we don't increase the size. I think it's better to just see all the options.